spec/factories/contacts.rb

```diff
@@ -16,5 +16,28 @@
   factory :contact do
     provider
     type { 'admin' }
+    name { Faker::Name.name }
+    email { Faker::Internet.email }
+    telephone { Faker::PhoneNumber.phone_number }
+  end
+
+  trait :admin_contact do
```

Comment:
maybe `:xxx_type` rather than `xxx_contact` as it object itself is a contact

Reply:
and done